src/storage/store/rel_table.cpp

```diff
@@ -245,24 +213,24 @@ void RelTable::initializeData(RelTableSchema* tableSchema) {
     bwdRelTableData->initializeData(tableSchema, wal);
 }
 
-vector<AdjLists*> RelTable::getAdjListsForNodeTable(table_id_t boundNodeTableID) {
+vector<AdjLists*> RelTable::getAllAdjLists(table_id_t boundTableID) {
     vector<AdjLists*> retVal;
-    if (fwdRelTableData->hasAdjLists(boundNodeTableID)) {
-        retVal.push_back(fwdRelTableData->getAdjLists(boundNodeTableID));
+    if (!fwdRelTableData->isSingleMultiplicity() && fwdRelTableData->isBoundTable(boundTableID)) {
```

Comment:
Up to u, but I guess we can still have isAdjLists(tableID) and is AdjColumn(tableID) API